cmd/main.go

```diff
@@ -338,16 +433,60 @@ Full docs can be found at github.com/aws/amazon-` + binName
 		}
 	}
 
+	// determine if user used a shorthand for sorting flag
+	sortFieldShorthandPath, ok := sortingKeysMap[*sortField]
+	if ok {
+		sortField = &sortFieldShorthandPath
+	}
```

Comment:
nit: reduce scope here:

```suggestion
	if sortFieldShorthandPath, ok := sortingKeysMap[*sortField]; ok {
		sortField = &sortFieldShorthandPath
	}